README.md

```diff
@@ -35,7 +35,7 @@ To install the API client library to your local Maven repository, simply execute
 Add this dependency to your project's build file:
 
 ```groovy
-compile "jp.ne.paypay:paypayopa:1.0.0"
+compile "jp.ne.paypay:paypayopa:0.1"
```

Comment:
on second thought let's make it 0.2.0 cause when we merge this PR, we would cut a new release and deploy to nexus with version `0.2.0
`

Reply:
Let's change this to 0.3.0

Reply:
@javidlulu